documentation/deliberately_vulnerable_flask_app/run_integration_tests.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import logging
```

Comment:
```suggestion
import logging
import os
```
Internal linter is complaining about the order of import. You can do this automatically in VScode https://donjayamanne.github.io/pythonVSCodeDocs/docs/refactoring_sort-imports/ or use some external tool like `isort`.  Other tools which may help you catch the linter issues ealier are `black` and `flake8`. It may be worth running them before submitting a diff and fix the issues surfaced in the new code you added :)